### ISSUE TITLE
--insertlines --removelines 

### ISSUE THREAD

**Issue by hyperknot** [nicklockwood/SwiftFormat#214]
I'm somewhat confused about the new rules and options supported by the newest 0.33.0 release.

TLDR: It's removing empty lines and it's not clear how to configure it not to.

Confusion 1. There are 4 rules but only 2 options to configure them: 

- blankLinesAtStartOfScope 
- blankLinesAtEndOfScope
- blankLinesBetweenScopes
- blankLinesAroundMark 

for options: `--insertlines` and `--removelines`.

Confusion 2. Both `--insertlines` and `--removelines` are enabled by default.

**Comment by nicklockwood** [comment]
Hi @hyperknot 

TLDR; to revert to the old behavior use `--disable blankLinesAtStartOfScope`

Re: Confusion 1

There is not a one-one mapping between rules and options. Several rules may be affected by the same option. In this case, both the `blankLinesAtStartOfScope` and `blankLinesAtEndOfScope` rule can be configured using the `--removelines` option.

All rules can be disabled explicitly by using `--disable <rule1>,<rule2>,etc`, but in some cases you can also disable a rule implicitly by using an option. In the case of the `blankLinesAtStartOfScope` rule, you can disable it by setting the `--removelines false` option, which disables all rules that remove lines.

Re: Confusion 2

The `--insertlines` and `--removelines` options are not mutually exclusive, which is why they can meaningfully both be enabled at once. The `--insertlines` option enables/disables any rule that inserts new lines (such as `blankLinesBetweenScopes` and `blankLinesAroundMark`), and the `--removelines` option enables/disables any rule that removes lines (such as `blankLinesAtStartOfScope` and `blankLinesAtEndOfScope`)

**Comment by hyperknot** [comment]
Thanks!
